packages/react-data-grid/src/ResizeHandle.js

```diff
@@ -1,23 +0,0 @@
-import React from 'react';
-
-import Draggable from './Draggable';
-
-const style = {
-  position: 'absolute',
-  top: 0,
-  right: 0,
-  width: 6,
-  height: '100%'
-};
-
-export default class ResizeHandle extends React.Component {
-  render() {
-    return (
-      <Draggable
-        {...this.props}
-        className="react-grid-HeaderCell__resizeHandle"
```

Comment:
We set the className prop here, but the prop wasn't used by Draggable, so I cleaned up some unused css.
This is probably why we do set the style prop. I moved those to the less file.